contracts/StakingPoolV2.sol

```diff
@@ -13,12 +14,14 @@ import '@openzeppelin/contracts/access/Ownable.sol';
 /// the difference between the user index and the current index by the user balance. User index and the pool
 /// index is updated and aligned with in the staking and withdrawing action.
 /// @author Elysia
-contract StakingPoolV2 is IStakingPoolV2, StakedElyfiToken, Ownable {
+contract StakingPoolV2 is IStakingPoolV2, StakedElyfiToken, Ownable, AccessControl {
   using StakingPoolLogicV2 for PoolData;
 
   constructor(IERC20 stakingAsset_, IERC20 rewardAsset_) StakedElyfiToken(stakingAsset_) {
     stakingAsset = stakingAsset_;
     rewardAsset = rewardAsset_;
+    _admin = msg.sender;
+    _setupRole(MANAGER_ROLE, msg.sender);
```

Comment:
Are there any way to revoke the manager role? It seems that there is no role admin for `MANAGER_ROLE` or `DEFAULT_ROLE_ADMIN`.